include/nuttx/lib/modlib.h

```diff
@@ -198,6 +198,8 @@ struct mod_loadinfo_s
   uintptr_t         datastart;   /* Start of.bss/.data memory in .text allocation */
   size_t            textsize;    /* Size of the module .text memory allocation */
   size_t            datasize;    /* Size of the module .bss/.data memory allocation */
+  unsigned int      textalign;   /* Necessary alignment of .text */
+  unsigned int      dataalign;   /* Necessary alignment of .bss/.text */
```

Comment:
should we use size_t like textsize/datasize?

Reply:
maybe. i don't think that large alignment is realistic though.

Reply:
fixed